website/pages/style.css

```diff
@@ -82,6 +82,237 @@
   color: white;
 }
 
+/* Print Styles - Hide Elements */
+@media print {
+  iframe,
+  .g-mega-nav,
+  .g-product-subnav,
+  .g-footer,
+  [aria-hidden='true'],
+  [id='sidebar'],
+  [id='edit-this-page'],
+  [id='jump-to-section'],
+  [id='__next-build-watcher'] {
+    display: none !important;
+  }
+}
+
+/* Print Styles - Page Spacing */
+@media print {
+  @page {
+    margin: 2cm 0.5cm;
+  }
+
+  @page :first {
+    margin-top: 0;
+  }
+
+  @page :last {
+    margin-top: 0;
+  }
+
+  blockquote {
+    break-inside: avoid;
+  }
+
+  body {
+    margin-bottom: 2cm;
+    margin-top: 2cm;
+  }
+
+  dl,
+  ol,
+  ul {
+    break-before: avoid;
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    break-after: avoid;
+    break-inside: avoid;
+  }
+
+  img {
+    break-inside: avoid;
+    break-after: avoid;
+  }
+
+  pre,
+  table {
+    break-inside: avoid;
+  }
+
+  pre {
+    white-space: pre-wrap;
+  }
+}
+
+@media print {
+  /* @todo: remove alongside @hashicorp/react-global-styles/_temporary-to-remove/layout.css */
+  .g-container {
+    max-width: 34em;
```

Comment:
`.g-container` has `32px` left and right `padding`, so the effective max-width for content is more like `30em` currently. It feels a bit tight to me so personally would add `padding: 0` here (though that's definitely an opinion rather than data-backed feedback 👌)

Reply:
Done. The measure was adjusted from `34em` to `33em` to follow the actual advice. A citation was also added.